### DISCUSSION TITLE
Upload image from pasted url

### DISCUSSION THREAD

**Discussion by DanielSpravtsev** [awcodes/filament-curator#370] [Ideas]
Hey @awcodes !
Is there a way to upload image from url pasting it to tiptap? Or maybe adding somewhere an action to paste url and then insert it to tiptap?

**Comment by awcodes** [comment]
Out of the box that isn't supported. Tiptap does have a paid extension that you could use though. 

https://tiptap.dev/api/extensions/file-handler
